docker/build_scripts/build.sh

```diff
@@ -147,18 +126,40 @@ rm -rf patchelf.tar.gz patchelf-$PATCHELF_VERSION
 
 ln -s $PY36_BIN/auditwheel /usr/local/bin/auditwheel
 
+# HACK: The newly compiled and installed curl messes with the system's
+# py2.6 installation, on which yum depends.  Work around it by
+# rewiring libcurl.so specifically for yum.  /usr/local/bin/ has higher
+# priority on the PATH than /usr/bin/
+cat <<'EOF' > /usr/local/bin/yum && chmod +x /usr/local/bin/yum
+#!/bin/bash
+if [ "x$(arch)" != xi686 ]; then
```

Comment:
This test should probably be more specific? So that it fails hard with other arch's like arm32.

Reply:
That's one of the reason I'd like to get #210 merged in, remove this hack once & for all. I'd rather do some rework on #210 which has some added value rather than rework this hack.